satzung.rst

```diff
@@ -50,11 +50,11 @@ Physik-Fachschaften (StAPF), die Vertrauenspersonen, das Kommunikationsgremium
 Die Wahlen von Mitgliedern des StAPF, des KomGrem und des TOPF sind
 Personenwahlen entsprechend der Geschäftsordnung der ZaPF.
 
-Die Mitgliedschaft im StAPF, dem Kommunikationsgremium oder dem TOPF endet mit
-Ablauf der Amtszeit, Ableben des Amtsinhabers oder der Amtsinhaberin,
-Niederlegung des Amtes oder Abwahl mit Zweidrittelmehrheit durch das Plenum. Der
-Antrag auf Abwahl ist bis 15:00 Uhr am Vortag bei der ausrichtenden Fachschaft
-anzukündigen.
+Die Mitgliedschaft im StAPF, den Vertrauenspersonen, dem Kommunikationsgremium
+oder dem TOPF endet mit Ablauf der Amtszeit, Ableben des Amtsinhabers oder der
```

Comment:
Statt Amtsinhaber und Amtsinhaber vielleicht "amtsinhabende Person"? Wenn wir das ganze in Zukunft geschlechtsgerecht formulieren wollen.

Reply:
Ja, das sollte man auch ändern, wenn #23 angenommen werden sollte, aber ich wollte die beiden Anliegen erst mal voneinander getrennt halten.

Reply:
Ah, ich sehe ich hatte genau den Formulierungsgedanken schonmal. :) Gut, dann warten wir damit noch.